### ISSUE TITLE
Bug tracking

### ISSUE THREAD

**Issue by shengjiex98** [radiolevity/verdant-shores#5]

**Comment by radiolevity** [comment]
That makes sense considering there is no user-input-error handling. I hadn't considered that to be 1.0 functionality (it's annoying but only ever caused by user error) so it wasn't a priority. It is any easy fix, however, but I think it might be worth making a specific task out of fixing all user inputs to handle erroneous inputs.

**Comment by shengjiex98** [comment]
I agree! Just bookkeeping here so we'll have a list of bugs and we know where to look for.

**Comment by radiolevity** [comment]
Ahhh that makes sense, I gotcha now. In that case, here's another bug: there is not currently a way to handle a reverse-time observation (where the end time is before the start time). It doesn't crash, but the observation should throw an error and not just attempt to record data anyway.

**Comment by shengjiex98** [comment]
Added to the list!
